diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6f2090ffa..c7eebbc99 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,6 +110,17 @@ option(SOUFFLE_ENABLE_TESTING "Enable/Disable testing" ${SOUFFLE_ENABLE_TESTING_
 option(SOUFFLE_GENERATE_DOXYGEN "Generate Doxygen files (html;htmlhelp;man;rtf;xml;latex)" "")
 option(SOUFFLE_CODE_COVERAGE "Enable coverage reporting" OFF)
 
+# Build with macosx 11
+if(APPLE)
+   set(CMAKE_THREAD_LIBS_INIT "-lpthread")
+   set(CMAKE_HAVE_THREADS_LIBRARY 1)
+   set(CMAKE_USE_WIN32_THREADS_INIT 0)
+   set(CMAKE_USE_PTHREADS_INIT 1)
+   set(THREADS_PREFER_PTHREAD_FLAG ON)
+   set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mmacosx-version-min=10.15")
+   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mmacosx-version-min=10.15")
+endif()
+
 # Add aditional modules to CMake
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
