From f1962b40aff1705a46ebfd29ddc3c9ab3919e519 Mon Sep 17 00:00:00 2001
From: Luc Tielen <luc.tielen@gmail.com>
Date: Sat, 4 Jun 2022 08:40:13 +0200
Subject: [PATCH] Add missing override keyword

---
 src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index 8f4ba085c..87636e56f 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -303,7 +303,7 @@ public:
         return Stream;
     }
 
-    bool endInput() {
+    bool endInput() override {
         const int Status = pclose(Stream);
         Stream = nullptr;
         if (Status == -1) {
-- 
2.25.1

